### PR TITLE
Add State Management Service

### PR DESCRIPTION
**Pull request by dontseyit**
This PR adds the SMS chart. The service is an API service and has MongoDB access.

### CODE REVIEW

File: charts/state-management/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# It is recommended to use it with quotes.
+appVersion: "1.1.0"
+
+# The major version of the API. Minor versions and patches are not relevant as they do not
+# introduce breaking changes.
+apiVersion: v2
+
+dependencies:
+- name: ghga-common
+  version: 0.2.8
```
Comment (ckaipf) [inline charts/state-management/Chart.yaml:22]:
```suggestion
  version: 0.2.10
```

File: charts/state-management/templates/serviceaccount.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```
Comment (ckaipf) [inline charts/state-management/templates/serviceaccount.yml:1]:
```suggestion
{{- include "ghga-common.serviceaccount" . }}
```

File: charts/state-management/templates/serviceaccount.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: {{ .Release.Name }}
```
Comment (ckaipf) [inline charts/state-management/templates/serviceaccount.yml:6]:
```suggestion
{{- include "ghga-common.serviceaccount" . }}
```

**Comment by ckaipf** [comment]
We have a new library chart release, we could test with this chart if everything is fine.

File: charts/state-management/templates/ingress.yml [hunk 4]
```diff
@@ -0,0 +1 @@
+{{- include "ghga-common.ingress" . }}
```
Comment (ckaipf) [inline charts/state-management/templates/ingress.yml:1]:
```suggestion
{{- include "ghga-common.ingress" . }}

```

File: charts/state-management/templates/mapping.yml [hunk 5]
```diff
@@ -0,0 +1 @@
+{{- include "ghga-common.mapping" . }}
```
Comment (ckaipf) [inline charts/state-management/templates/mapping.yml:1]:
```suggestion
{{- include "ghga-common.mapping" . }}

```

**Comment by dontseyit** [comment]
> We have a new library chart release, we could test with this chart if everything is fine. After applying the changes we need to run `helm dep up`. @ckaipf

I ran the `helm dep up` and I've published the updated **Chart.lock** here: https://github.com/ghga-de/charts/pull/83/commits/472caf5f98a6ad866dfa84da58e337c6777294a5

**Comment by dontseyit** [comment]
I'm not sure the reason of the linting error

**Review by ckaipf — APPROVED** [review]

**Pull request merged**